react/features/invite/components/add-people-dialog/native/AddPeopleDialog.js

```diff
@@ -159,6 +170,38 @@ class AddPeopleDialog extends AbstractAddPeopleDialog<Props, State> {
             _dialOutEnabled
         } = this.props;
         const { inviteItems, selectableItems } = this.state;
+        const { navigation, t, theme } = this.props;
+        const { palette } = theme;
+
+        navigation.setOptions({
+            headerLeft: () => (
+                <TouchableWithoutFeedback
```

Comment:
Looks like we could maybe have some helper BacknavigationButton component? I've seen this code a couple of times now...

Reply:
Created a separate component.